packages/launch-config/src/debug-config/config.ts

```diff
@@ -8,6 +8,64 @@ const testFlpSandboxHtml = 'test/flpSandbox.html';
 const indexHtml = 'index.html';
 const testFlpSandboxMockServerHtml = 'test/flpSandboxMockServer.html';
 
+/**
+ * Returns the `migratorMockIntent` with a leading `#` if it doesn't already start with one.
+ * If the input is undefined, it will return undefined.
+ *
+ * @param {string} [migratorMockIntent] - The optional mock intent string to be used in the migrator.
+ * @returns {string | undefined} - The migrator mock intent prefixed with `#` or undefined.
+ */
+export function getMigratorMockIntentWithHash(migratorMockIntent?: string): string | undefined {
+    if (!migratorMockIntent) {
+        return undefined;
+    }
+    return migratorMockIntent.startsWith('#') ? migratorMockIntent : `#${migratorMockIntent}`;
+}
+
+/**
+ * Determines the parameters to use for the mock HTML file based on the given inputs.
+ * If the `targetMockHtmlFile` is `testFlpSandboxMockServerHtml` and `migratorMockIntentWithHash` is provided,
+ * it will return `migratorMockIntentWithHash`. Otherwise, it defaults to `flpAppIdWithHash` or empty string.
+ *
+ * @param {string | undefined} targetMockHtmlFile - The target mock HTML file.
+ * @param {string | undefined} [migratorMockIntentWithHash] - The migrator mock intent.
+ * @param {string | undefined} [flpAppIdWithHash] - The FLP app ID with a leading `#`.
+ * @returns {string} - The parameters to use, either `migratorMockIntentWithHash` or `flpAppIdWithHash`, defaults to an empty string.
+ */
+export function getParamsForMockHtml(
+    targetMockHtmlFile: string | undefined,
+    migratorMockIntentWithHash?: string,
+    flpAppIdWithHash?: string
+): string {
+    if (targetMockHtmlFile === testFlpSandboxMockServerHtml && migratorMockIntentWithHash) {
+        return migratorMockIntentWithHash;
+    }
+    return flpAppIdWithHash ?? '';
```

Comment:
```suggestion
`    return migratorMockIntentWithHash ?? flpAppIdWithHash ;`
```
Not sure that we need to strictly check value to `targetMockHtmlFile` . Can't we accept what the user passes in?
Also, could we simplify the function to this.

Reply:
Good point 👍 
Thanks ✅